### DISCUSSION TITLE
Broken link on play.battlesnake.com

### DISCUSSION THREAD

**Discussion by Vam-Jam** [BattlesnakeOfficial/feedback#248] [Bug Reports]
https://play.battlesnake.com/account/battlesnakes `What are engine regions?` links to `https://docs.battlesnake.com/guides/tips/engine-regions`, 

It should be `https://docs.battlesnake.com/guides/engine-regions`

**Comment by bvanvugt** [comment]
Oooo yes, thanks for reporting this! I've updated the code, it'll get fixed on next deploy!
